types/web-element.d.ts

```diff
@@ -187,6 +187,8 @@ export interface ScopedElement extends Element, PromiseLike<WebElement> {
 
   rightClick(): Promise<WebElement>;
 
+  isSelected(): Promise<boolean>
```

Comment:
`isSelected` would returns an instance of `ScopedValue`.

Reply:
updated in  91b5ec3